File: programs_for_beginers/Ascending_order.cpp
#include <iostream>

using namespace std;

int main()
{   int temp, j, i;
    int A[5]={5,3,7,1,8};
for(i=0; i<5; i++){
    for(j=0; j<5-i; j++){
        if(A[j+1]<A[j]){temp=A[j];
                        A[j]=A[j+1];
                        A[j+1]=temp;}

    }
    cout<<A[i]<<" ";
}
}

File: Python_programming_for_beginners/ascending_order.py
'''Ascending numbers'''
numbs=[]
size=int(input("enter size:  "))
for i in range(size):
         no=int(input("enter a no:  "))
         numbs.append(no)
for i in range(size):
    for j in range(i+1):   
        if(numbs[i]<numbs[j]):
            temp=numbs[i]
            numbs[i]=numbs[j]
            numbs[j]=temp
print("new array is: ", numbs)
